tests/basic_env.py

```diff
@@ -20,7 +20,9 @@ class TestDir(TestCase):
     DATA_SUB_CONTENTS = DATA_SUB
     FOO = 'foo'
     FOO_CONTENTS = FOO
-    BAR = 'bar'
+    # len(BAR) must != len(FOO). MacOs time resolution prevents successful
+    # non-checksum file update checks, thus failing tests on Mac
+    BAR = 'barr'
```

Comment:
Let's modify BAR_CONTENTS instead by appending additional 'r' to it, no need to change the file name itself, since it is the contents that we care about.